templates/commands/goldentest/record_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package goldentest implements golden test related subcommands.
+package goldentest
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+
+	"github.com/abcxyz/abc/templates/common"
+	"github.com/abcxyz/pkg/logging"
+	"github.com/abcxyz/pkg/testutil"
+	"github.com/google/go-cmp/cmp"
+)
+
+//nolint:paralleltest // This test cannot be run in parallel
```

Comment:
Could you add more explanation to this comment explaining why it can't run in parallel?

Reply:
Removed - now it can be paralleled. Previously they use the same temp dir.